deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+destination_repo=${1:-${GITHUB_REPO}}
+build_dir=styleguide
+target=18f-pages
+
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "GITHUB_TOKEN is required." >&2
+    exit 1
+fi
+
+# Clean
+rm -rf $build_dir
+
+# Build
+npm run build
+
+# Git init
+cd $build_dir
+git --version
```

Comment:
Yeah, similar to how travis dumps out a bunch of information about node, python, etc.